internal/pkg/daemon/bpfrecorder/bpf/recorder.bpf.c

```diff
@@ -24,6 +25,13 @@ struct {
     __type(value, char[MAX_COMM_LEN]);  // command name
 } comms SEC(".maps");
 
+struct {
+    __uint(type, BPF_MAP_TYPE_HASH);
+    __uint(max_entries, MAX_MNTNS_LEN);
+    __type(key, u32);                   // PID
+    __type(value, u64);                 // mntns ID
+} system_mntns SEC(".maps");
```

Comment:
I guess we can just use a single rodata variable rather than a full map here, right? 

Example: 
https://github.com/iovisor/bcc/blob/dfb0d98c8f03a4701839503ca29c7d4986196b88/libbpf-tools/syscount.bpf.c#L17
https://github.com/iovisor/bcc/blob/dfb0d98c8f03a4701839503ca29c7d4986196b88/libbpf-tools/syscount.c#L423

Reply:
No I mean we can determine the mount namespace and pass it into the ebpf application before starting it.

Reply:
Is there any way to pass parameters before the ebpf program starts? From libbpf go,I didn't find some related function,such as bpf.NewModuleFromBufferArgs(*args).